README.md

```diff
@@ -11,13 +11,12 @@ Check the [`old-versions`](https://github.com/nodecg/awesome-nodecg/blob/master/
 NodeCG is a broadcast graphics framework and application. It provides an API and structure for making dynamic graphical assets for broadcast use. NodeCG graphics packages are called "bundles". NodeCG is primarily aimed at [Twitch](http://www.twitch.tv) broadcasters using [Open Broadcaster Software](https://obsproject.com), but is usable in any environment that can render HTML, including [CasparCG](http://casparcg.com).
 
 Still not clear on what NodeCG is and what it is used for? These videos go over some of the more prominent uses of NodeCG:
-- [Tip of the Hats 2015 Graphics Overview](https://www.youtube.com/watch?v=Z4-qaya5-0Y)
 - [Tip of the Hats 2016 Graphics Overview](https://www.youtube.com/watch?v=F0xGN-aSytA)
 - [Awesome Games Done Quick 2016 Graphics Overview](https://www.youtube.com/watch?v=oAzj9Zddogs)

```

Comment:
I think the AGDQ2016 overview should be removed, as it tends to raise more questions than it answers due to its unusual use of Electron as the renderer.